manifests/n/Nvidia/CUDA/12.4.1/Nvidia.CUDA.installer.yaml

```diff
@@ -4,25 +4,26 @@
 PackageIdentifier: Nvidia.CUDA
 PackageVersion: 12.4.1
 Platform:
-- Windows.Desktop
+  - Windows.Desktop
 InstallerType: portable

```

Comment:
The installer is not a portable installer

Reply:
Alright, I'll close this PR in favor of yours that has change the InstallerType, I didn't even notice it was falsely set to portable